points.yml

```diff
@@ -9,15 +9,18 @@
   link:
     href: http://example.com
     caption: Find out more!
-- name: All the Pythons
-  ready: false
+- name: Multiple Pythons
```

Comment:
Just a small note: I feel the original name was better, more enthusiastic. "Multiple Pythons" sounds a little bit dry.